enhancements/sig-architecture/15-resourcebasedscheduling/README.md

```diff
@@ -0,0 +1,221 @@
+# Resource Based Scheduling
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [website](https://github.com/open-cluster-management-io/website/)
+
+## Summary
+
+The proposed work would enhance the Placement APIs to support scheduling based on the resource usage/capacity of managed clusters.
+
+## Motivation
+
+During placement scheduling, managed clusters will be selected based on resource(s) (like memory and cpu etc.) usage/capacity to avoid scheduling workloads on clusters with insufficient resources.
+
+Link to the feature request: https://github.com/open-cluster-management-io/community/issues/52
+
+### User Stories
+
+#### Story 1: User is able to select clusters with specific resource(s), whose allocatable to capacity ratio is larger.
+  - User wants to deploy an application on a cluster whose CPU usage is less than others;
+  - User wants to deploy an application on 2 clusters whose allocatable to capacity ratio of both CPU and memory is larger than others;
+
+#### Story 2: User is able to select clusters with more allocatable specific resource(s) by specifying the resource(s) in placement.
+  - User wants to deploy an application on a cluster with largest memory allocatable;
+
+#### Story 3: User is able to select clusters based on some resource(s) usage and keep the decisions adjusted according to the usage changes.
+  - A CPU-intensive application would like to be deployed on the cluster with least CPU utilization rate.
+
+#### Story 4: User is able to select clusters based on some resource(s) usage once, without considering the usage changes afterwards, keeping the decisions pinned.
+  - Some critical service should run on a steady set of clusters, even if the resource usage of those clusters changes;
+
+#### Story 5: Support scheduling based on usage/allocatable of resources other than CPU and Memory. For example,
+  - AI applications are expected to run on clusters with least gpu usage;
+  - An application can be scheduled to cluster with largest available network bandwidth;
+
+### Goals
+
+- Sotry 1 and 2, allow user to select managed clusters based on resource usage/capability with the Placement APIs;
+
+### Non-Goals
+- Balance workload across the fleet based on cluster resource usage;
+- Story 3 and 4 are related to churning policy of placement, and will be covered in a separated enhancement proposal;
```

Comment:
is this different then that the steady weighting?

Reply:
Yes, they are related. Weight of `steady` plugin should be adjusted automatically according to the churning policy of a placement. And we may also support some advanced features, like `churningSeconds`, to describe to what extent we can stand for the cluster churning.